holmes/plugins/toolsets/mongodb/mongodb.py

```
@@ -33,7 +33,9 @@ def _parse_json_param(value: str, param_name: str) -> Any:
     try:
         return json.loads(value)
     except json.JSONDecodeError as e:
-        raise ValueError(f"Invalid JSON for '{param_name}': {e}. Received: {value[:200]}")
+        raise ValueError(
+            f"Invalid JSON for '{param_name}': {e}. Received: {value[:200]}"
+        )
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)